File: 프로그래머스/level1_서울에서 김서방 찾기.py
def solution(seoul):
    answer = ''
    for i in range(len(seoul)):
        if seoul[i] == "Kim":
            answer = "김서방은 " + str(i) + "에 있다"
    return answer

print(solution(["Jane", "Kim"]))

# return "김서방은 {}에 있다".format(seoul.index('Kim'))

File: 프로그래머스 복습(java)/src/level1/level1_서울에서_김서방_찾기.java
package level1;

public class level1_서울에서_김서방_찾기 {
    static class Solution {
        public String solution(String[] seoul) {
            for (int i = 0; i < seoul.length; i++) {
                if (seoul[i].equals("Kim")) {
                    return "김서방은 " + i + "에 있다";
                }
            }
            return "";
        }
    }

    public static void main(String[] args) {
        Solution solution = new Solution();
        String[] seoul = {"Jane", "Kim"};
        System.out.println(solution.solution(seoul));
    }
}
